src/components/NewsCard/NewsCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import { CircularProgress } from '@mui/material';
+
+import styles from './NewsCard.module.scss';
+
+function NewsCard({
+  news,
+  onClick,
+  isLoading = false,
+}) {
+  const defaultImg = '/assets/images/defaultImg.png';
```

Comment:
use it in place or move outside the component - and change name - it's a constant